### DISCUSSION TITLE
Obsidian 自动同步功能功能  最新版本只能手动完成sync

### DISCUSSION THREAD

**Discussion by Bealliant** [windingwind/zotero-better-notes#1235] [Q&A]
大佬您好，想请问一下这个sync功能好像有点问题...
![image](https://github.com/user-attachments/assets/64a45aa7-ccb7-4754-8d5f-db982428074f)
我设置了sync =2, 然后使用`export current file`好像无法自动更新 只能完成手动更新

我又下载了旧的版本`realease -1.14 Beta 66`https://github.com/windingwind/zotero-better-notes/releases/tag/v1.1.4-beta.66
发现自动同步是可以运行的

**Comment by Bealliant** [comment]
p.s. 新版本的better notes和老版本相比，老版本点zotero的链接可以跳转并且显示，但是新版本点链接跳转之后是无法显示选中的文段

**Comment by windingwind** [comment]
> 然后使用export current file好像无法自动更新 只能完成手动更新

请详细描述，建议报告为issue并附截图/录屏/debug output

**Comment by Liuy811** [comment]
请问解决了吗
